packages/tupaia-web/src/api/queries/useMapOverlayData.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useQuery } from 'react-query';
+import { get } from '../api';
+import { EntityCode, ProjectCode, SingleMapOverlayItem } from '../../types';
+
+export const useMapOverlayData = (
+  projectCode?: ProjectCode,
+  entityCode?: EntityCode,
+  mapOverlayCode?: SingleMapOverlayItem['code'],
+) => {
+  return useQuery(
+    ['mapOverlayData', projectCode, entityCode, mapOverlayCode],
+    async () => {
+      return get(
+        `measureData?mapOverlayCode=${mapOverlayCode}&organisationUnitCode=${entityCode}&projectCode=${projectCode}&shouldShowAllParentCountryResults=${
```

Comment:
it would be great to use the axios params field here for readability and ease of maintenance. 
Also Ethan has re-done the endpoints for the map overlays so this should use `legacyMapOverlayReport` or `report` depending on whether or not it is legacy.

Reply:
Is that endpoint for fetching the measure data, or for something else?

Reply:
yes that's right. for measure data

Reply:
I'm currently getting an error using this, and it seems to be different to `measureData` which is for map overlays (is that right?) and reports which I was using for dashboard report data. I'll add this in as part of part 2

Reply:
oh yeah it looks like there is a bug in the endpoint so yes that's leave this as is for now